### DISCUSSION TITLE
problemi pagamento utente

### DISCUSSION THREAD

**Discussion by patrizio1995** [woocommerce/woocommerce#51597] [Checkout Flow]
Buona sera, vi scrivo perche abbiamo dei problemi discutibili con i pagamenti. 
confido in una vostra soluzione 
![errore](https://github.com/user-attachments/assets/2d3cad09-2267-431d-ac69-1df13071a68b)

**Comment by alexflorisca** [comment]
Hi there, the best place to ask for support is the [WooCommerce forums](https://wordpress.org/support/plugin/woocommerce/), Please post there and provide a bit more detail so that someone may be able to help you.
